docs/Getting-Started/Beginning-Asterisk.md

```diff
@@ -49,7 +49,7 @@ Resources for understanding
 
 * Installing and Configuring Asterisk
 + [Asterisk: The Definitive Guide](http://www.asteriskdocs.org/)   3rd Edition
-+ [The Asterisk Wiki](//)
++ [The Asterisk Wiki](/)
```

Comment:
Why the change back? We are referring to this site as the documentation site going forward, since it's not strictly a wiki.